### ISSUE TITLE
Angular Material Support Documentation 

### ISSUE THREAD

**Issue by manojgudi2016** [oferh/ng2-completer#390]
Hi Team,

You have done great job in making this re-usable component and sharing with the community, thank you for all the hard work and support.

Could you make some documentation on how this component will work with Angular Material, I am planning to use this control with Angular Material. 

Thank you,
Manoj

**Comment by oferh** [comment]
@manojgudi2016 just out of curiosity if your'e using Material why not use Material autocomplete or select component?

**Comment by manojgudi2016** [comment]
@oferh : You are right, I already started using Angular Material Autocomplete, wasn't aware of it when I asked the question. Thanks for the reply. But this is definitely a great control for native angular projects.
